src/aere_theme.erl

```diff
@@ -150,7 +150,7 @@ render(Theme, ThemedTxt = {themed, _, _}) ->
     render(Theme, [ThemedTxt]);
 render(Theme, ThemedTxts) when is_list(ThemedTxts) ->
     AnsiStr = lists:flatten(lists:map(fun(T) -> apply_theme(Theme, T) end, ThemedTxts)),
-    string:trim(AnsiStr, both, unicode_util:whitespace()).
+    string:trim(AnsiStr, trailing, unicode_util:whitespace()).
```

Comment:
What's the problem with trimming both sides? Indention or something else?

Reply:
Yeah, indentation gets problematic in the first line. It's hard to handle when coloring characters are involved.